### PR TITLE
adding dataset members

### PR DESCRIPTION
**Pull request by Nike682631**
Signed-off-by: nikunj goyal <nikunjgoyal31@gmail.com>

### CODE REVIEW

**Comment by Nurkambay** [comment]
#6

File: src/main/java/ListDatasetsTest.java [hunk 1]
```diff
@@ -28,10 +27,16 @@ public static void main(String[] args) throws IOException {
...
 
-    private static void tstListDsn(ZOSConnection connection, String dataSetName) throws IOException {
+    private static void tstListMembers(ZOSConnection connection, String dataSetName) {
+        ListParams params = new ListParams.Builder().build();
+        List<Dataset> datasets = zosfiles.List.listMembers(connection, dataSetName, params);
+        datasets.forEach(LOG::info);
+    }
+    private static void tstListDsn(ZOSConnection connection, String dataSetName) {
         ListParams parms = new ListParams.Builder().build();
         List<Dataset> datasets = zosfiles.List.listDsn(connection, dataSetName, parms);
```
Comment (frankgiordano) [inline src/main/java/ListDatasetsTest.java:41]: remove zosfiles.  import it.

Comment (frankgiordano) [reply src/main/java/ListDatasetsTest.java:41]: I renamed List to ZosList to avoid conflict with java.util.List

Comment (frankgiordano) [reply src/main/java/ListDatasetsTest.java:41]: see #36

**Comment by frankgiordano** [comment]
@Nike682631 hi i think we can merge these changes now? what do you think?

**Comment by Nike682631** [comment]
Sure, I'll be adding download datasets but i can do that in another pr so
feel free to merge them.

**Pull request merged**